pkg/pgmodel/query_builder.go

```diff
@@ -53,6 +53,11 @@ const (
 	AND time >= '%[4]s'
 	AND time <= '%[5]s'
 	GROUP BY s.id`
+
+	// Jan 1st, 1970
+	millisecondMin = 0
```

Comment:
I don't love limiting us to 1970 and 3000. There will def be use cases that want to insert at least older data I'd suggest setting 
```
minTime = time.Unix(math.MinInt64/1000+62135596801, 0)
maxTime = time.Unix(math.MaxInt64/1000-62135596801, 999999999)
```